dashboard/head.py

```diff
@@ -117,9 +117,10 @@ def __init__(self, http_host, http_port, http_port_retries, redis_address,
         ip, port = redis_address.split(":")
         self.gcs_client = connect_to_gcs(ip, int(port), redis_password)
         self.server = aiogrpc.server(options=(("grpc.so_reuseport", 0), ))
+        grpc_ip = "127.0.0.1" if self.ip == "127.0.0.1" else "0.0.0.0"
         self.grpc_port = ray._private.tls_utils.add_port_to_grpc_server(
-            self.server, "[::]:0")
-        logger.info("Dashboard head grpc address: %s:%s", self.ip,
+            self.server, f"{grpc_ip}:0")
+        logger.info("Dashboard head grpc address: %s:%s", grpc_ip,
```

Comment:
Is the bind ip the same as `--dashboard-host` in https://github.com/ray-project/ray/blob/master/python/ray/scripts/scripts.py#L374?

Reply:
seems may not be the case: `self.ip = ray.util.get_node_ip_address()`